src/linear_algebra/cayley_hamilton.lean

```diff
@@ -0,0 +1,98 @@
+/-
```

Comment:
Hmm... how about renaming this file `char_poly`? I think that's what people would look for. And also, we'll probably want to add more facts later: roots are eigenvalues, simp-lemma for char_poly of 1x1 matrix, etc.

Reply:
Done.